translations/monero-core_de.ts

```diff
@@ -998,17 +998,17 @@
     <message>
         <location filename="../pages/Settings.qml" line="267"/>
         <source>Password</source>
-        <translation type="unfinished">Passwort</translation>
+        <translation>Passwort</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="302"/>
         <source>Layout settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Layout-Einstellungen</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="319"/>
         <source>Custom decorations</source>
-        <translation type="unfinished"></translation>
+        <translation>Angepasste Oberfläche</translation>
```

Comment:
On my Mac the checkbox just switches between the "regular" native window title bar and replaces it with a "custom" black title bar. Nothing else happens as far as I can see, you cannot even chose or specify - just accept what's already done / built-in or use the regular window title bar:

![bildschirmfoto 2017-03-30 um 10 51 38](https://cloud.githubusercontent.com/assets/26762624/24495916/feefcb84-1536-11e7-8525-816e39c194c4.png)

![bildschirmfoto 2017-03-30 um 10 51 50](https://cloud.githubusercontent.com/assets/26762624/24495914/fc11b5a8-1536-11e7-88b2-7289f8a3ae91.png)

Maybe in other OS more magic happens with that checkbox?